src/js/node/net.ts

```diff
@@ -1445,8 +1452,23 @@ function listenInCluster(
   onListen,
 ) {
   exclusive = !!exclusive;
+  // shim until SocketAddress gets added. lazy-loaded b/c isIP is quite expensive.
+  const getConnKeyShim = (() => {
+    let key;
+    return () => {
+      if (!key) {
+        key = isIP(hostname) + ":" + hostname + ":" + port;
+      }
+      return key;
+    };
+  })();
 
   if (cluster === undefined) cluster = require("node:cluster");
+  Object.defineProperty(server, "_connectionKey", {
```

Comment:
If we're going to shim this then we should make sure the enumerable / writable property is set if necessary